# This contains the motor commands for a 6 motor air vehicle. (0.0 - 1.0)
float32 m1
float32 m2
float32 m3
float32 m4
float32 m5
float32 m6
